accounts/migrations/0044_new-analytics-permissions.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2019-10-23 16:02
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("accounts", "0043_gestational-age-can-be-blank")]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="child",
+            options={
+                "ordering": ["-birthday"],
+                "permissions": [
+                    ("can_view_child_analytics_data", "Can View Child Analytics Data")
+                ],
+            },
+        ),
+        migrations.AlterModelOptions(
+            name="demographicdata",
+            options={
+                "ordering": ["-created_at"],
+                "permissions": [
+                    (
+                        "can_view_demographics_analytics_data",
+                        "Can View Demographics Analytics Data",
+                    )
+                ],
+            },
+        ),
+        migrations.AlterModelOptions(
+            name="organization",
+            options={
+                "ordering": ["name"],
+                "permissions": (
+                    ("can_view_organization", "Can View Organization"),
+                    ("can_edit_organization", "Can Edit Organization"),
+                    ("can_create_organization", "Can Create Organization"),
+                    ("can_remove_organization", "Can Remove Organization"),
+                    ("can_view_experimenter", "Can View Experimenter"),
+                    ("can_view_analytics", "Can View Analytics"),
+                ),
+            },
+        ),
+        migrations.AlterModelOptions(
+            name="user",
+            options={
+                "ordering": ["id"],
+                "permissions": (
+                    ("can_create_users", "Can Create User"),
+                    ("can_view_users", "Can View User"),
+                    ("can_edit_users", "Can Edit User"),
+                    ("can_remove_users", "Can Remove User"),
+                    ("can_view_user_permissions", "Can View User Permissions"),
+                    ("can_edit_user_permissions", "Can Edit User Permissions"),
+                    ("can_read_all_user_data", "Can Read All User Data"),
+                    ("can_read_usernames", "Can Read User Usernames"),
+                    ("can_view_user_analytics_data", "Can View User Analytics Data"),
```

Comment:
we don't use can_view_user_analytics_data, correct? It seems like it might be complicated anyway because the pivot table uses demographic, child, & response info - not actually anything directly stored on the user model anyway, and the demo/child/response info aren't especially separable

Reply:
We don't; see initial PR description/comment - I just put these here in case we wanted to leverage them later. We could theoretically use them (this one in particular, for, say, precluding certain users from seeing registration data) but you're probably right in that it's not likely. I'll nix them.